### PR TITLE
Throw a parse error if a comma missing between two alter operations

### LINKED ISSUE #189: ALTER TABLE ... CHANGE ... CHANGE ...;
``` sql
ALTER TABLE `tb_foo`
CHANGE `inmsg` `inmsg` date NULL AFTER `outmsg2`
CHANGE `inmsg2` `inmsg2` time NULL AFTER `inmsg`;
```

Current version don't see missed comma between two `CHANGE`.

### PR DESCRIPTION
**Pull request by devenbansod**
Fix #189 

Signed-off-by: Deven Bansod <devenbansod.bits@gmail.com>

### CODE REVIEW

File: src/Components/AlterOperation.php [hunk 1]
```diff
@@ -276,4 +287,17 @@ public static function build($component, array $options = array())
 
         return $ret;
     }
+
+    /**
+     * Check if token's value is one of the common keywords
+     * between column and table alteration
+     *
+     * @param String $token_value Value of current token
```
Comment (williamdes) [inline src/Components/AlterOperation.php]: `String` lowercase ?

Comment (devenbansod) [reply src/Components/AlterOperation.php]: @williamdes thanks for pointing it out. I've updated it and force-pushed. 👍

**Comment by devenbansod** [comment]
@williamdes any idea why the `elseif` line is being shown as not covered in the coverage report?

**Comment by williamdes** [comment]
@devenbansod Yes, go ahead, okay for me :+1:

**Comment by devenbansod** [comment]
@williamdes it seems like it was just due to improper formatting. I moved the first condition of the elseif statement to previous line and now it's covering the entire diff. :-)

**Pull request merged**